src/test/java/com/browserstack/utils/browserstack/BrowserStackUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.browserstack.utils.browserstack;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class BrowserStackUtil {
+
+    private static final String BROWSERSTACK_USER_ENV = "BROWSERSTACK_USER";
+    private static final String BROWSERSTACK_ACCESS_KEY_ENV = "BROWSERSTACK_ACCESSKEY";
+    private static final String BROWSERSTACK_HUB_URL_FORMAT = "https://%s:%s@hub.browserstack.com/wd/hub";
+
+    private final String browserstackUserName;
+    private final String browserstackAccessKey;
+
+    public BrowserStackUtil() {
+        this.browserstackUserName = System.getenv(BROWSERSTACK_USER_ENV);
+        this.browserstackAccessKey = System.getenv(BROWSERSTACK_ACCESS_KEY_ENV);
```

Comment:
what if these environment vars are accidentally not set ?
The README template suggests this: Alternatively, you can also hardcode username and access_key objects in the caps.json file.

in your case, do we need a separate CSV file or can we identify a way if the environment vars are not set?